### DISCUSSION TITLE
Custom tooltip values

### DISCUSSION THREAD

**Discussion by wiggitamoo** [flant/grafana-statusmap#225] [Q&A] [answered]
Awesome plugin, thanks for the hard work! I'm using this plug-in with an SQL database. Is it possible to pass in additional text columns from a query to be displayed in a tooltip on hover?

**Comment by wiggitamoo** [comment, marked as answer]
I was able to figure out how to pass in custom text to the tooltips by utilizing the "**Split Query Alias**". By passing in the following value as the metric. 

``` sql
SELECT <Y_Label> || '|' || <URL_Param> AS metric
```

I split the alias using the delimiter `|`. Then I can reference `${__y_label_0}` in the "**Label Template**" field and `${__y_label_1}` in the URL field.

**Comment by ppajersk** [comment]
@wiggitamoo Could you expand on how you made this work? I am attempting to graph workflow executions in a time series graph, and when hovering over a datapoint, it would be pretty handy to see additional information about the workflow, like number of failures/successes.

These metrics would just be integer values so I don't think it's impossible to do it, just not sure where Split Query Alias and Label Template would be located.
